1-org/envs/shared/backend.tf

```diff
@@ -16,7 +16,7 @@
 
 terraform {
   backend "gcs" {
-    bucket = "UPDATE_ME"
+    bucket = "bkt-prj-b-seed-tfstate-fd64"
```

Comment:
Was this change intentional?

Reply:
no!! I included it in the commit by mistake, sorry